app/javascript/packages/document-capture/services/upload.tsx

```diff
@@ -1,22 +1,21 @@
 import { FormError } from '@18f/identity-form-steps';
 
-/** @typedef {import('../context/upload').UploadSuccessResponse} UploadSuccessResponse */
-/** @typedef {import('../context/upload').UploadErrorResponse} UploadErrorResponse */
-/** @typedef {import('../context/upload').UploadFieldError} UploadFieldError */
+import type {
+  UploadSuccessResponse,
+  UploadErrorResponse,
+  UploadFieldError,
+  UploadImplementation,
+} from '../context/upload';
 
 export class UploadFormEntryError extends FormError {
-  /** @type {string} */
   field = '';
 }
 
 export class UploadFormEntriesError extends FormError {
-  /** @type {UploadFormEntryError[]} */
-  formEntryErrors = [];
+  formEntryErrors = [] as UploadFormEntryError[];
```

Comment:
I think it's generally preferable to try to avoid casting in favor of an explicit type on the property if possible, since casting can sometimes bypass checks we might want.

```suggestion
  formEntryErrors: UploadFormEntryError[] = [];
```